### PR TITLE
Move conda tests into own lanes

### PR DESCRIPTION
**Pull request by bast**
So that timeouts do not color the entire test set red. Later we can have a closer look at the timeouts.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -80,7 +99,14 @@ before_install:
...
 
 install:
-  - travis_wait 30 ./testing/dependencies/install_dependencies.sh
+  - ./testing/dependencies/travis/install.sh
+  - ./testing/dependencies/travis/cmake.sh
+  - ./testing/dependencies/travis/ninja.sh
+  - if [[ "$ANACONDA_TESTS_ONLY" == 1 ]]; then
+      ./testing/dependencies/travis/anaconda.sh;
+    else
+      ./testing/dependencies/travis/eigen.sh;
```
Comment (robertodr) [inline .travis.yml]: Should this always be installed?

File: .travis.yml [hunk 2]
```diff
@@ -80,7 +99,14 @@ before_install:
   - mkdir -p $HOME/Downloads $HOME/Deps
 
 install:
-  - travis_wait 30 ./testing/dependencies/install_dependencies.sh
+  - ./testing/dependencies/travis/install.sh
+  - ./testing/dependencies/travis/cmake.sh
+  - ./testing/dependencies/travis/ninja.sh
+  - if [[ "$ANACONDA_TESTS_ONLY" == 1 ]]; then
+      ./testing/dependencies/travis/anaconda.sh;
```
Comment (robertodr) [inline .travis.yml]: If Anaconda install is cached, then it's not a huge problem to always run the installation, rather than have this test.

Comment (bast) [reply .travis.yml]: Good point I will remove the if-check.

File: .travis.yml (hunk 1 above)
Comment (bast) [reply .travis.yml]: Not sure I understand the question but I guess it will answer itself if I remove the if-check.

**Comment by bast** [comment]
I will spawn an issue for the last comment: #353.

File: .travis.yml (hunk 1 above)
Comment (robertodr) [reply .travis.yml]: Yeah, it answered itself :smile:

**Comment by robertodr** [comment]
LO AND BEHOLD! The greenery!!!!

**Comment by bast** [comment]
Brings tears to my eyes :-)

**Pull request merged**